src/frontend/main.go

```diff
@@ -96,6 +103,37 @@ func InitTracerProvider() *sdktrace.TracerProvider {
 	return tp
 }
 
+func InitMeterProvider() (metric.MeterProvider, error) {
+	ctx := context.Background()
+
+	client := otlpmetricgrpc.NewClient(
+		otlpmetricgrpc.WithInsecure(),
+		otlpmetricgrpc.WithEndpoint("otelcol:4317"))
```

Comment:
Does the Go SDK provide a default port (4317)?

Reply:
I added the environment `OTEL_EXPORTER_OTLP_ENDPOINT` to service frontend, the metrics sdk will use this environment by default.